### PR TITLE
Add screening icon on Home and Library pages

### PR DESCRIPTION
**Pull request by KvnMrs**
Issue : https://github.com/blockframes/blockframes/issues/7770

### CODE REVIEW

File: libs/event/src/lib/components/request-screening/request-screening.component.html [hunk 1]
```diff
@@ -16,7 +16,8 @@
       <mat-spinner diameter="24" *ngIf="status === 'sending'"></mat-spinner>
     </ng-container>
     <ng-template #icon>
-      <mat-icon svgIcon="ask_screening_2" [matTooltip]="screeningRequest[status]"></mat-icon>
+      <mat-icon *ngIf="status === 'available'" svgIcon="ask_screening_2" [matTooltip]="screeningRequest[status]"></mat-icon>
+      <mat-icon *ngIf="status === 'sent'" svgIcon="ask_screening_2" [matTooltip]="screeningRequest[status]" color='disabled'></mat-icon>
```
Comment (phpgeekfr) [inline libs/event/src/lib/components/request-screening/request-screening.component.html:20]: this should work
```suggestion
      <mat-icon svgIcon="ask_screening_2" [color]="status === 'sent' ? 'disabled': ''"></mat-icon>
```

Comment (phpgeekfr) [reply libs/event/src/lib/components/request-screening/request-screening.component.html:20]: `[matTooltip]="screeningRequest[status]"` should be on the button 
```
  <ng-container *ngIf="iconOnly else button">
    <button mat-icon-button class="translucent" [disabled]="status !== 'available'" [matTooltip]="screeningRequest[status]" (click)="requestScreening()">
      <ng-container *ngTemplateOutlet="icon"></ng-container>
    </button>
  </ng-container>
  <ng-template #button>
    <button mat-button class="translucent" [disabled]="status !== 'available'" [matTooltip]="screeningRequest[status]" (click)="requestScreening()">
      <ng-container *ngTemplateOutlet="icon"></ng-container>
      {{ screeningRequest[status] }}
    </button>
  </ng-template>
```

Like this we will have the tooltip on the text also
image

File: libs/event/src/lib/components/request-screening/request-screening.component.html [hunk 2]
```diff
@@ -16,7 +16,8 @@
       <mat-spinner diameter="24" *ngIf="status === 'sending'"></mat-spinner>
```
Comment (phpgeekfr) [inline libs/event/src/lib/components/request-screening/request-screening.component.html:16]: remove ng-container and do : 
```suggestion
      <mat-spinner *ngIf="status === 'sending'; else icon" diameter="24"></mat-spinner>
```

Comment (phpgeekfr) [reply libs/event/src/lib/components/request-screening/request-screening.component.html:16]: and spinner is not well centered 
image

File: libs/event/src/lib/components/request-screening/request-screening.component.html (hunk 1 above)
Comment (KvnMrs) [reply libs/event/src/lib/components/request-screening/request-screening.component.html:20]: Yes, but like this i dont get the tooltip when the color is disabled

Comment (phpgeekfr) [reply libs/event/src/lib/components/request-screening/request-screening.component.html:20]: you can wrap buttons in a `span` to correct issue 
```
  <span [matTooltip]="screeningRequest[status]">
    <ng-container *ngIf="iconOnly else button">
      <button mat-icon-button class="translucent" [disabled]="status !== 'available'" (click)="requestScreening()">
        <ng-container *ngTemplateOutlet="icon"></ng-container>
      </button>
    </ng-container>
    <ng-template #button>
      <button mat-button class="translucent" [disabled]="status !== 'available'" (click)="requestScreening()">
        <ng-container *ngTemplateOutlet="icon"></ng-container>
        {{ screeningRequest[status] }}
      </button>
    </ng-template>
  </span>
  ```
  
  https://stackoverflow.com/questions/61402969/no-tool-tip-is-shown-in-google-chrome-when-the-button-is-disabled-in-angular

File: libs/event/src/lib/components/request-screening/request-screening.component.html [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
+        {{ screeningRequest[status] }}
+      </button>
+    </ng-template>
+  </span>
 
   <ng-template #icon>
-    <ng-container *ngIf="status === 'sending'; else icon">
-      <mat-spinner diameter="24" *ngIf="status === 'sending'"></mat-spinner>
-    </ng-container>
+    <mat-spinner diameter="24" *ngIf="status === 'sending'; else icon" diameter="24"></mat-spinner>
```
Comment (phpgeekfr) [inline libs/event/src/lib/components/request-screening/request-screening.component.html:17]: you have two `diameter="24`

**Review by phpgeekfr — APPROVED** [review]

**Pull request merged**